prime-dotnet-webapi/ApiDbContext.cs

```diff
@@ -260,6 +264,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .WithMany(s => s.BusinessLicences)
                 .HasForeignKey(bl => bl.SiteId);
 
+            modelBuilder.Entity<SelfDeclarationDocument>()
```

Comment:
These manual configurations are not needed. EF configures this automatically with Convention Over Configuration (by calling the foreign keys EnrolleId for example)